speech/google/cloud/speech/sample.py

```diff
@@ -112,3 +117,180 @@ def encoding(self):
         :returns: String value of Encoding type.
         """
         return self._encoding
+
+    def async_recognize(self, language_code=None, max_alternatives=None,
+                        profanity_filter=None, speech_context=None):
+        """Asychronous Recognize request to Google Speech API.
```

Comment:
There's a bunch of ways to package the data up and pass it around but I can't think of any that are good.